tile-json/map.js

```diff
@@ -0,0 +1,162 @@
+import { Loader } from '@googlemaps/js-api-loader';
+import axios from 'axios';
+import { GOOGLE_MAPS_TOKEN } from '@env';
+
+/**
+ * This example shows how to use the GeoJSOn response from our Vector Tile Server with Google Maps
+ *
+ * To establish a connection with Chargetrip GraphQL API you need to have an API key.
+ * The key sin this example is are public ones and gives access only to a part of our extensive database.
```

Comment:
The keys in this example are public and only provide access to a small part of our extensive database.